### DISCUSSION TITLE
MoonRay for Cinema 4D?

### DISCUSSION THREAD

**Discussion by walissonftz** [dreamworksanimation/openmoonray#6] [Q&A]
Is there a possibility to implement for Cinema 4D??

**Comment by randypacker** [comment]
Hi Walissonftz, it would be nice, though we suspect the community may get there before us.   Let's see!

**Reply by heavyrain266** [reply]
I imagine that it won't be easy, to my knowledge C4D does not support Hydra natively nor have a plugin made by community.
